programs/oeis/002/A002487.asm

```diff
@@ -0,0 +1,19 @@
+; A002487: Stern's diatomic series (or Stern-Brocot sequence): a(0) = 0, a(1) = 1; for n > 0: a(2*n) = a(n), a(2*n+1) = a(n) + a(n+1).
```

Comment:
For hand-written programs, you can include "Coded manually .." in the second line as done also by Antti: https://github.com/ckrause/loda/pull/48/files
This will prevent that the maintenance job overwrites your comments.

Reply:
ok, I have done that.